test/linalg.jl

```diff
@@ -202,6 +202,48 @@ StaticArrays.similar_type(::Union{RotMat2,Type{RotMat2}}) = SMatrix{2,2,Float64,
         @test normalize(SVector(1,2,3), 1) ≈ normalize([1,2,3], 1)
         @test normalize!(MVector(1.,2.,3.)) ≈ normalize([1.,2.,3.])
         @test normalize!(MVector(1.,2.,3.), 1) ≈ normalize([1.,2.,3.], 1)
+
+        # nested vectors
+        a  = SA[SA[1, 2], SA[3, 4]]
+        a′ = convert(Vector{Vector{Int}}, a)
```

Comment:
That apostrophe looks almost like adjoint, could you change `a′` to `a2`, or `av` or something?

Reply:
Sure, done.